src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -71,17 +70,41 @@ public void init() throws ServletException {
   // TODO: Check if URL is valid.
   public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    Optional<String> optionalVideoUrl = getParameter(request, LINK_INPUT);
-    Optional<Entity> existingEntity = checkUrlInDatabase(optionalVideoUrl);
+    if (request.getParameter(PARAM_NAME) == null) {
+      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Missing name parameter.");
+      return;
+    }
+    String lectureName = request.getParameter(PARAM_NAME);
+
+    if (request.getParameter(PARAM_LINK) == null) {
+      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Missing link parameter.");
```

Comment:
nit: please update the error messages to be constants

Reply:
Done, thanks!